sdl-core/pom.xml

```diff
@@ -340,6 +340,25 @@
 			<version>${json4s.version}</version>
 		</dependency>
 
+		<!-- Dependency for StatusInfoRestApi-->
```

Comment:
spark-core includes an own jetty version under package "org.sparkproject.jetty.servlet".

And Hadoop-client includes jetty dependencies with scope "provided" (mostly)
```
[INFO] +- org.apache.hadoop:hadoop-client:jar:3.3.1:compile
[INFO] |  +- org.apache.hadoop:hadoop-common:jar:3.3.1:compile
...
[INFO] |  |  +- org.eclipse.jetty:jetty-servlet:jar:9.4.43.v20210629:provided
[INFO] |  |  |  +- org.eclipse.jetty:jetty-security:jar:9.4.43.v20210629:provided
[INFO] |  |  |  \- org.eclipse.jetty:jetty-util-ajax:jar:9.4.43.v20210629:provided
...
[INFO] |  +- org.apache.hadoop:hadoop-yarn-client:jar:3.3.1:compile
[INFO] |  |  +- org.eclipse.jetty.websocket:websocket-client:jar:9.4.40.v20210413:compile
[INFO] |  |  |  +- org.eclipse.jetty:jetty-client:jar:9.4.43.v20210629:provided
[INFO] |  |  |  |  \- org.eclipse.jetty:jetty-http:jar:9.4.43.v20210629:provided
[INFO] |  |  |  +- org.eclipse.jetty:jetty-util:jar:9.4.43.v20210629:provided
[INFO] |  |  |  +- org.eclipse.jetty:jetty-io:jar:9.4.40.v20210413:compile
[INFO] |  |  |  \- org.eclipse.jetty.websocket:websocket-common:jar:9.4.40.v20210413:compile
[INFO] |  |  |     \- org.eclipse.jetty.websocket:websocket-api:jar:9.4.40.v20210413:compile
...
```

sdl-parent pom has dependency mgmt for the following jetty dependencies:
```
			<!-- Shaded deps marked as provided. These are promoted to compile scope
                 in the modules where we want the shaded classes to appear in the
                 associated jar. -->
			<dependency>
				<groupId>org.eclipse.jetty</groupId>
				<artifactId>jetty-http</artifactId>
				<version>${jetty.version}</version>
				<scope>provided</scope>
			</dependency>
			<dependency>
				<groupId>org.eclipse.jetty</groupId>
				<artifactId>jetty-continuation</artifactId>
				<version>${jetty.version}</version>
				<scope>provided</scope>
			</dependency>
			<dependency>
				<groupId>org.eclipse.jetty</groupId>
				<artifactId>jetty-servlet</artifactId>
				<version>${jetty.version}</version>
				<scope>provided</scope>
			</dependency>
			<dependency>
				<groupId>org.eclipse.jetty</groupId>
				<artifactId>jetty-servlets</artifactId>
				<version>${jetty.version}</version>
				<scope>provided</scope>
			</dependency>
			<dependency>
				<groupId>org.eclipse.jetty</groupId>
				<artifactId>jetty-proxy</artifactId>
				<version>${jetty.version}</version>
				<scope>provided</scope>
			</dependency>
			<dependency>
				<groupId>org.eclipse.jetty</groupId>
				<artifactId>jetty-client</artifactId>
				<version>${jetty.version}</version>
				<scope>provided</scope>
			</dependency>
			<dependency>
				<groupId>org.eclipse.jetty</groupId>
				<artifactId>jetty-util</artifactId>
				<version>${jetty.version}</version>
				<scope>provided</scope>
			</dependency>
			<dependency>
				<groupId>org.eclipse.jetty</groupId>
				<artifactId>jetty-security</artifactId>
				<version>${jetty.version}</version>
				<scope>provided</scope>
			</dependency>
			<dependency>
				<groupId>org.eclipse.jetty</groupId>
				<artifactId>jetty-plus</artifactId>
				<version>${jetty.version}</version>
				<scope>provided</scope>
			</dependency>
			<dependency>
				<groupId>org.eclipse.jetty</groupId>
				<artifactId>jetty-server</artifactId>
				<version>${jetty.version}</version>
				<scope>provided</scope>
			</dependency>
			<dependency>
				<groupId>org.eclipse.jetty</groupId>
				<artifactId>jetty-webapp</artifactId>
				<version>${jetty.version}</version>
				<scope>provided</scope>
			</dependency>
```

I agree to use the version provided by hadoop-client. Its also good to mention these dependencies here, otherwise they would be listed as undeclared but used dependencies by the corresponding maven check plugin.
But we should use Dependency mgmt and remove the version tag.

Reply:
OK, 2 out of 3 dependencies that I use we used in the parent-pom, so I just used ${jetty.version}.
Not sure what to do with this dependency though
`		<dependency>
			<groupId>org.glassfish.jersey.media</groupId>
			<artifactId>jersey-media-json-jackson</artifactId>
			<version>2.34</version>
		</dependency>`
Should it also be mentioned in the parent pom?

Reply:
I would leave this like that. The parent pom handles versions from spark-dependencies and libraries used by several sdl-modules.